js/layout-javascript/agenda-code.js

```diff
@@ -2498,6 +2498,7 @@ DynamicList.prototype.addDetailViewData = function(entry) {
   var _this = this;
 
   if (_.isArray(entry.entryDetails) && entry.entryDetails.length) {
+    _this.Utils.Record.fetchImageContent(_this, entry);
```

Comment:
With `fetch` I'd expect a return of the fetched content. Since this assigns the image content directly the function should be named accordingly.

```suggestion
    _this.Utils.Record.assignImageContent(_this, entry);
```

Reply:
Fixed.